framework/doc/content/source/problems/ReferenceResidualProblem.md

```diff
@@ -7,23 +7,37 @@
 By default, MOOSE checks convergence using relative and absolute criteria. Once the residual drops
 below either an absolute tolerance, or the residual divided by the initial residual for the current
 time step drops below a relative tolerance, the solution is considered converged. This works well for
-many problems, but in cases where the solution changes minimally between time steps, the initial
-residual can be very small, making the relative convergence check much too stringent.
+many problems, but there are some scenarios that are problematic for convergence:
 
-The `ReferenceResidualProblem` checks for convergence by comparing the residual to a different
+1. There is a large scaling difference between the solution variables, so taking the $l^2$ norm of the
+combined vector skews the convergence check so that it becomes increasingly dominated by the variable
+with larger absolute values in the residual vector as the scaling difference increases.
+
+2. The model conditions have changed very little from the previous step, so the initial residual is very
+low because the solution is essentially converged before any nonlinear iterations have been performed.
+The standard convergence check that uses the initial residual as a reference quantity eventually fails
+after a few steps because the solution is forced to converge to tigheter limits with each time
```

Comment:
tigheter -> tighter

Reply:
Fixed. We really need to do something like run aspell on the .md files in our pre-commit hooks.